lib/storage/metadata/mongoclient/MongoClientInterface.ts

```diff
@@ -3124,7 +3177,13 @@ class MongoClientInterface {
     putBucketIndexes(bucketName: string, indexSpecs, log: werelogs.Logger, cb: ArsenalCallback<void>) {
         const c = this.getCollection<ObjectMetastoreDocument>(bucketName);
         const indexes = MongoUtils.indexFormatObjectToMongoArray(indexSpecs);
-        c.createIndexes(indexes).then(() => cb(null)).catch(err => {
+        c.createIndexes(indexes).then(result => {
```

Comment:
same, may be OK if the (same) index already exists, and we just have a race condition on creating the same index....

Reply:
I ignore errors here, it's true that in this case we can ignore, as the index is already there.